hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -292,4 +292,10 @@ private OMConfigKeys() {
   public static final long OZONE_OM_ADMIN_PROTOCOL_WAIT_BETWEEN_RETRIES_DEFAULT
       = 1000;
 
+  /**
+   * Default threshold configurations for OMLockMetrics.
+   */
+  public static final String OZONE_OM_LOCK_REPORTING_THRESHOLD_MS_KEY =
+      "ozone.om.lock.reporting.threshold.ms";
```

Comment:
> TestOzoneConfigurationFields : This test is failing.

Please add the newly introduced configs to ozone-default.xml, because the above test is doing assertion for the configs.


  <property>
    <name>ozone.om.lock.reporting.threshold.ms</name>
    <value>3000</value>
    <tag>OZONE, MANAGEMENT</tag>
    <description>
    ....add description about the config....
    </description>
  </property>

Reply:
Thank you, I have added the newly introduced configuration to ozone-default.xml.